packages/client/test/miner/pendingBlock.spec.ts

```diff
@@ -90,12 +89,15 @@ const setup = () => {
 }
 
 describe('[PendingBlock]', async () => {
-  const originalValidate = BlockHeader.prototype['_consensusFormatValidation']
-  BlockHeader.prototype['_consensusFormatValidation'] = td.func<any>()
-  td.replace<any>('@ethereumjs/block', { BlockHeader })
+  BlockHeader.prototype['_consensusFormatValidation'] = vi.fn()
+  vi.doMock('@ethereumjs/block', () => {
+    {
+      BlockHeader
+    }
+  })
 
-  const originalSetStateRoot = DefaultStateManager.prototype.setStateRoot
-  DefaultStateManager.prototype.setStateRoot = td.func<any>()
+  // const originalSetStateRoot = DefaultStateManager.prototype.setStateRoot
```

Comment:
Comment?

Reply:
I must have missed that commented-out line. It's been removed.